cross/mingw-w64/Portfile

```diff
@@ -18,9 +18,9 @@ long_description    Mingw-w64 is an advancement of the original mingw.org projec
 
 homepage            http://mingw-w64.sourceforge.net/
```

Comment:
Nitpick: can you ensure that links in all your changes are HTTPS instead of HTTP?